### ISSUE TITLE
Bug for service provisioning with multiple bindings

### ISSUE THREAD

**Issue by SteveZhang-CDL** [IBM/cloud-operators#96]
I provision a cloudant service with multiple bindings according to the following yaml file:

```
apiVersion: ibmcloud.ibm.com/v1alpha1
kind: Service
metadata:
  name: my-cloudant
spec:
  plan: standard 
  serviceClass: cloudantnosqldb
---
apiVersion: ibmcloud.ibm.com/v1alpha1
kind: Binding
metadata:
  name: my-binding-cloudant-1
spec:
  serviceName: my-cloudant
---
apiVersion: ibmcloud.ibm.com/v1alpha1
kind: Binding
metadata:
  name: my-binding-cloudant-2
spec:
  serviceName: my-cloudant
```

However,  I can see there are two service instances in IBM Cloud Portal resource list, one has two bindings, another one has none, showing as below:

<img width="1304" alt="cloudant" src="https://user-images.githubusercontent.com/4101246/76293615-57057200-62ec-11ea-983b-0d5d34c909c9.png">

<img width="1114" alt="cloudant1" src="https://user-images.githubusercontent.com/4101246/76293782-9f249480-62ec-11ea-9723-234414afed04.png">

<img width="1146" alt="cloudant2" src="https://user-images.githubusercontent.com/4101246/76293802-aa77c000-62ec-11ea-89a8-686e6ec165bf.png">

**Comment by vazirim** [comment]
This bug is a logical data race that seems to happen if you have all 3 resources in one file and submit them at once. For now, to prevent this you can submit the cloudant service first, and have another file with the bindings that you submit next.

**Comment by vazirim** [comment]
Incidentally, the operator knows that there was a problem, and has a message in its log saying that the state of the world may be inconsistent. It tries to delete the cloudant but the deletion does not work because the service is inactive initially and rejects deletion.

**Comment by cdl-zhuail** [comment]
@vazirim Thank you for your reply,  according to your recommendation, it seems that I have to make sure that the service is ready before I provision its bindings. Is that correct?

**Comment by vazirim** [comment]
@cdl-zhuail I have a bug fix for this issue, I will make a release shortly (in open-source).

**Comment by vazirim** [comment]
Fixed in v0.1.5

**Comment by SteveZhang-CDL** [comment]
@vazirim Thank you so much!
